android/BOINC/app/src/main/java/edu/berkeley/boinc/rpc/PlatformInfo.java

```diff
@@ -64,4 +64,12 @@ public PlatformInfo createFromParcel(Parcel in) {
         }
     };
 
+    @SuppressWarnings("java:S115")
```

Comment:
What this warning is about? I don't like to suppress warning. Is it possible to have no warning here?

Reply:
It's a warning provided by SonarLint. It relates to the naming of constant variables.